lib/models/entity.py

```diff
@@ -14,6 +18,61 @@ def __init__(self, name: str, description: str, damage: int):
         super().__init__(name, description)
 
 
+class Room(Entity):
+
+    class Exit(Entity):
```

Comment:
Why make these subclasses? It means when you're defining them, or referring to them, you have to access them as `Room.Exit`

It also means the class _definition_ is redefined every time a `Room` is instantiated. We're not at the point where these types of performance issues matter, but it's worth knowing about at least

Reply:
Fixed.